### ISSUE TITLE
[css-easing-2] Simplify the syntax of `<linear-stop-list>`

### ISSUE THREAD

**Issue by yarusome** [w3c/csswg-drafts#8932]
[2.1.1 Syntax](https://drafts.csswg.org/css-easing-2/#linear-easing-function-syntax) defines `<linear-stop-list>` as:

```
<linear-stop-list> = [ <linear-stop> ]#
```

and Step 3 of [2.1.2 Parsing](https://drafts.csswg.org/css-easing-2/#linear-easing-function-parsing) states that:

> If there are less than two items in *stopList*, then return *failure*.

Comparing to the [syntax](https://drafts.csswg.org/css-images-4/#color-stop-syntax) of `<color-stop-list>`:

```
<color-stop-list> =
  <linear-color-stop> , [ <linear-color-hint>? , <linear-color-stop> ]#
```

it seems more straightforward to define that:

```
<linear-stop-list> = [ <linear-stop> ]#{2,}
```

and drop Step 3 mentioned above.

**Comment by cdoublev** [comment]
Assuming there cannot be grammar failure after this change, I would also rename *2.1.2 Parsing* to *2.1.2 Create a linear easing function*.

**Comment by cdoublev** [comment]
Fixed by aaad9c7: a single "stop" point is now valid (as well as in a gradient color stop list).
